snap/hooks/install

```diff
@@ -23,8 +23,8 @@ function create_file_structure () {
     dir_copy_if_not_exists "modules" "${SNAP_DATA}" 550
     dir_copy_if_not_exists "plugins" "${SNAP_DATA}" 770
 
-    dir_copy_if_not_exists "config" "${SNAP_COMMON}" 770
-    add_folder "${SNAP_COMMON}/config/certificates" 770
+    dir_copy_if_not_exists "config" "${SNAP_DATA}" 770
+    add_folder "${SNAP_DATA}/config/certificates" 770
```

Comment:
wait...this should be `$SNAP_DATA/etc/opensearch/configuration_file` but I understand this path fixing will be done in a separate PR

Reply:
Yes, I'm tracking it [here](https://warthogs.atlassian.net/browse/DPE-1464)